README.md

```diff
@@ -596,6 +596,18 @@ $role->givePermissionTo('edit articles');
 $role->revokePermissionTo('edit articles');
 ```
 
+### DecryptException
+
+Working with a lot of permissions, your cache can grow quickly. When using the database cache driver, you might get a `DecryptException`. In that case, just change your cache migration file from:
+
+```php
+$table->text('value');
+```
+to:
+```php
+`$table->longText('value');
```

Comment:
You're talking about the migration suggested by Laravel itself, right? 

ie: https://github.com/laravel/framework/blob/5.4/src/Illuminate/Cache/Console/stubs/cache.stub

... and nothing to do with the Spatie laravel-permission package, right?